CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased Changes
 * Added color schemes for documentation based on user preference ([#290](https://github.com/Roblox/roact/pull/290)).
 * Fixed stack trace level when throwing an error in `createReconciler` ([#297](https://github.com/Roblox/roact/pull/297)).
+* Optimized the memory usage of 'createSignal' implementation.
```

Comment:
Can you add a link to the PR like the other items in the list?

Reply:
Done.